example.js

```diff
@@ -6,41 +6,133 @@ import React, { Component } from 'react'
 import { Map as LeafletMap, TileLayer } from 'react-leaflet'
 import Control from 'react-leaflet-control'
 import { Browser } from 'leaflet'
-import {ReactChoropleth, ReactDot, scales} from './lib'
-import {createGrid} from 'browsochrones'
+import { ReactGridMapLayer, interpolators, colorizers, classifiers } from './lib'
+import { createGrid } from 'browsochrones'
 import { render } from 'react-dom'
 import { scaleLog } from 'd3-scale'
 
-const DOT = 'dot'
-const CHOROPLETH_QUANTILE = 'choropleth quantile'
-const CHOROPLETH_EQUAL = 'choropleth equal'
-const CHOROPLETH_LOG = 'choropleth log'
 const NYC_HUDSON_STREET = [40.73535, -74.00630]
 // const KC_HOSPITAL_HILL = [39.08333, -94.575]
 
 export default class GridualizerExample extends Component {
+
+  deStijlColors = [
```

Comment:
Meaning anything that's not mutable object instance state should be defined at the top level?